pg_mon.c

```diff
@@ -958,8 +966,12 @@ pg_mon(PG_FUNCTION_ARGS)
                     ArrayType  *arry;
                     int n = 0, idx = 0;
                     for (n = 0; n < MAX_TABLES && entry->bitmap_scans[n] != 0; n++)
-                            datums[idx++] = ObjectIdGetDatum(&entry->bitmap_scans[n]);
-                    arry = construct_array(datums, idx, OIDOID, sizeof(Oid), false, 'i');
+                        datums[idx++] = ObjectIdGetDatum(entry->bitmap_scans[n]);
+#if PG_VERSION_NUM >= 160000
```

Comment:
that part cannot be refactored out, e.g. to a separate function ? looks very repetitive

Reply:
maybe, can you propose it here or in a separate PR?